src/test/ui-fulldeps/internal-lints/lint_pass_impl_without_macro.rs

```diff
@@ -17,7 +17,8 @@ declare_lint! {
 
 struct Foo;
 
-impl LintPass for Foo { //~ERROR implementing `LintPass` by hand
+impl LintPass for Foo {
+    //~^ERROR implementing `LintPass` by hand
```

Comment:
This (and below) was the one place were I think the formatting was slightly better originally. But it's not too bad. IMHO, the best to fit in with the formatting would be if we had a way to point downward at error lines. But we don't.